Add Alerts System to nRF Cloud (#9920)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

    * MCUboot child image files to properly access external flash on newer nRF9160DK versions.
    * An :file:`overlay_mcuboot_ext_flash.conf` file to enable MCUboot use of external flash.

>     * Sending an alert to the cloud on boot as well as when a temperature limit is exceeded. 

Reviewer: ```suggestion
    * Sending an alert to the cloud on boot and when a temperature limit is exceeded. 
```
To nRF Cloud?

* :ref:`nrf_cloud_rest_device_message` sample:

>   * Added sending an alert to the cloud on boot.

Reviewer: to nRF Cloud?

  * Removed location events and event handlers.

* :ref:`lib_nrf_cloud_alerts` library:

>   * Added new library for sending notifications of critical device events to nRF Cloud, using either REST or MQTT connections.

Reviewer: ```suggestion
  * A new library for sending notifications of critical device events to nRF Cloud, using either REST or MQTT connections.
```
